### PR TITLE
Move from travis to github actions

### PR DESCRIPTION
**Pull request by serge-sans-paille**

### CODE REVIEW

**Comment by serge-sans-paille** [comment]
@marimeireles not in a hurry, but this one seems ready :-)

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  build:
+    runs-on: ubuntu-18.04
```
Comment (marimeireles) [inline .github/workflows/workflow.yml:12]: Should we test on ubuntu only?

**Review by marimeireles — COMMENTED** [review]
Seems good :)
Other than the OS thing can't think of anything

File: .github/workflows/workflow.yml (hunk 1 above)
Comment (serge-sans-paille) [reply .github/workflows/workflow.yml:12]: I think it's fine - memestra is OS agnostic.

**Pull request merged**